source/_integrations/speedtestdotnet.markdown

```diff
@@ -1,50 +1,39 @@
 ---
-title: Speedtest.net
+title: SpeedTest.net
```

Comment:
The `t` is not capitalized in their brand, it is the reason why the tests on the documentation fail as well. Please adjust it.